contents/handbook/who-we-are-building-for.md

```diff
@@ -0,0 +1,92 @@
+
+> Example: PostHog anytime from their Series B to IPO
+
+Marketing and customer success should primarily focus on this ICP, but should also develop **high-*potential* customers** - customers that are likely to later become high-growth customers (e.g. James and Tim during YC). We should be in maintenance mode for **hobbyists**, such as engineers building side projects. We want to be the first tool that technical founders add to their product.
+
+Why? We believe that the best tech companies are increasingly engineering-led. By building for the best product-engineers we'll create a positive feedback loop of being the best product for them. This is clearly differentiated from others in the space which are focused on less technical PMs. As more of the leading tech companies adopt being engineering-led and having highly technical PMs PostHog will become the clear product leader.
+
+## Our current ICP
+
+|  | High-growth startup |
+| --- | --- |
+| **Description** | Startups that have product-market fit and are quickly scaling up with new customers, hiring, and adding more revenue. |
+| **Criteria** | - 15-500 employees<br />- $100k+/month in revenue OR very large number of consumer users<br />- Raised from leading investors<br />- Not yet IPO’ed |
+| **Why they matter?** | - Able to efficiently monetize them<br />- Very quick sales cycle<br />- Act as key opinion leaders for earlier-stage startups/slower moving companies<br />- Strong opinions on what they need - helping us build a better product |
+| **Job role** | We build for the power users of the **The Product team**<br /><br />**Primary focus**<br/><br/>- Product-engineers (full-stack engineer skewing front-end)<br />- PMs (those that are highly technical)<br /><br />**Should be usable by**:<br />- Designers<br />- Less technical PMs<br />- Marketers<br />|
+| **Examples** | PostHog anytime from their Series B to IPO, Linear, Ramp, Vercel, Retool |
+
+Each team will focus more or less on different members of the product team. This is detailed on their team pages.
+
+## FAQs
+
+### Why double-down on product engineers vs. making it more accessible and easy to use?
+
+**Background**
+
+- We have good product-market fit with engineers and ok fit with less technical product managers.
+- Most of the existing product analytics tools are built for product managers who are less technical, not engineers.
+- However, the best tech company are increasingly engineering led, with all members of the product team being technical. We believe more companies will work like this in future.
+
+**Principles**
+
+- We should double-down on the best product-engineers at the best product companies.
+- Going incredibly far with a more specific user type instead of placating every body, means that we'll get over the threshold for "I need to tell my engineer friends about this". We only get word-of-mouth growth by doing a remarkable job, not an average one for lots of people.
```

Comment:
```suggestion
- Going incredibly far with a more specific user type instead of placating every body means that we'll get over the threshold for "I need to tell my engineer friends about this". We only get word-of-mouth growth by doing a remarkable job, not an average one for lots of people.
```